dist/common/scripts/scylla_setup

```diff
@@ -283,16 +283,16 @@ if __name__ == '__main__':
         cur_version=out('scylla --version', exception=False)
         if len(cur_version) > 0:
             if is_debian_variant():
-                new_version=out('sudo -u scylla {}/scylla-housekeeping --uuid-file /var/lib/scylla-housekeeping/housekeeping.uuid --repo-files \'/etc/apt/sources.list.d/scylla*.list\' version --version {} --mode i'.format(scriptsdir(), cur_version), shell=True, exception=False)
+                new_version=out('sudo -u scylla {}/scylla-housekeeping --uuid-file /var/lib/scylla-housekeeping/housekeeping.uuid --repo-files \'/etc/apt/sources.list.d/scylla*.list\' version --version {} --mode i'.format(scriptsdir(), cur_version), shell=True, exception=False, timeout=60)
```

Comment:
Can you please extract `timeout` into a local variable or a constant so if we decide to change 60 to something else, we don't need to scan the whole file?

Reply:
@penberg thanks for the review, I have forced-pushed a newer version